src/sbvr-api/sbvr-utils.ts

```diff
@@ -372,12 +372,60 @@ export const isModelNew = async (
 	return !cachedIsModelNew.has(modelName);
 };
 
+const bindsForAffectedIds = (
+	bindings: AbstractSQLCompiler.Binding[],
+	request?: Pick<
+		uriParser.ODataRequest,
+		| 'vocabulary'
+		| 'abstractSqlModel'
+		| 'method'
+		| 'resourceName'
+		| 'affectedIds'
+	>,
+) => {
+	if (!request || !request.affectedIds) {
```

Comment:
I assume this would work:
```suggestion
	if (request?.affectedIds == null) {
```
?

Reply:
It does, though I'm puzzled as to why `===` leads to type errors